CefSharp.Core/Internals/CefTaskScheduler.h

```
@@ -42,7 +43,11 @@ namespace CefSharp
                 }
                 catch (Exception^)
                 {
-
+                    // This should never ever happen.
```

Perhaps just remove the `try/catch` and let the exception bubble?